src/engine/macroInt.cpp

```diff
@@ -245,6 +245,77 @@ void DivMacroInt::mask(unsigned char id, bool enabled) {
 #undef CONSIDER_OP
 #undef CONSIDER
 
+void DivMacroInt::retrig(unsigned char id)
```

Comment:
Pleasefixformatting,or I will rewrite thisfunction.

Reply:
Can't you just run linter over it after merge?

Reply:
Fine! I will rewrite the darn function I guess...

Reply:
I don't care as long as the effect functionality stays the same

Reply:
Then don't complain about my code style either :p